src/Serilog.Sinks.Loggly/Sinks/Loggly/DurableLogglySink.cs

```diff
@@ -33,26 +33,32 @@ class DurableLogglySink : ILogEventSink, IDisposable
             long? eventBodyLimitBytes,
             LoggingLevelSwitch levelControlSwitch,
             long? retainedInvalidPayloadsLimitBytes,
-            IFormatProvider formatProvider = null)
+            int? retainedFileCountLimit = null,
+            IFormatProvider formatProvider = null
+            )
         {
             if (bufferBaseFilename == null) throw new ArgumentNullException(nameof(bufferBaseFilename));
 
+            //use a consistent UTF encoding with BOM so no confusion will exist when reading / deserializing
+            var encoding = new UTF8Encoding(encoderShouldEmitUTF8Identifier:false);
+
             //handles sending events to Loggly's API through LogglyClient and manages the pending list
             _shipper = new HttpLogShipper(
                 bufferBaseFilename, 
                 batchPostingLimit, 
                 period, 
                 eventBodyLimitBytes, 
                 levelControlSwitch,
-                retainedInvalidPayloadsLimitBytes);
+                retainedInvalidPayloadsLimitBytes,
+                encoding);
 
             //writes events to the file to support connection recovery
             _sink = new RollingFileSink(
                 bufferBaseFilename + "-{Date}.json",
                 new LogglyFormatter(formatProvider), //serializes as LogglyEvent
                 bufferFileSizeLimitBytes,
-                null,
-                Encoding.UTF8);
```

Comment:
Hmm looks like the encoding was already being explicitly specified, which should have avoided the issue I pointed to.

Changing this now to a `false` will break existing buffer files.